lib/order.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'twilio_api'
+
+class Order
+  attr_reader :order, :dishes
+
+  def initialize(menu)
+    @order = Hash.new(0)
```

Comment:
Avoid naming instance variables the same as the class name. Otherwise you risk asking class instances this sort of thing:

```ruby
order = Order.new
# Hey order, what's your...order?
order.order # :( weird
```